en/editor/mainMenu/index.md

```diff
@@ -1,156 +1,155 @@
-# 主菜单
+# Main Menu
```

Comment:
@slackmoehrle Have you reviewed this one?

Reply:
yes